code/app/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,7 @@
         android:name="android.hardware.camera"
         android:required="false" />
     <uses-permission android:name="android.permission.CAMERA"/>
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
external storage is needed then taking gallery photo, please keep it

Reply:
Fixed now